### PR TITLE
Web Accessibility

### PR DESCRIPTION
**Pull request by Kwamay**
-[ ] Page title - Changed to "Portfolio"
-[ ] Image text alternatives - Added image text alternatives to the image tag
-[ ] Text headings - Added <h1-h6> tags to some text in the project
-[ ] Color contrast- No issues found
-[ ] Resize - No issues found
-[ ] Interaction - No issues found
-[ ] Moving content - No issues found
-[ ] Multimedia - No issues found
-[ ] The basic structure of the page - No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
-    <title>Document</title>
+    <title>Portfolio</title>
 </head>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] Please adequately and briefly describes the content of the page title. For example:

**Poor titles:**
- Microverse project | Your portfolio.
- Portfolio project

**Good titles:**
- Mark's portfolio
- Mark full-stack web developer

File: index.html [hunk 2]
```diff
@@ -51,15 +51,15 @@ <h4>LET'S CONNECT</h4>
...
-                    <h1>Tonic</h1>
+                    <h2>Tonic</h2>
                     <div class="frame">
                         <ul>
                             <li class="frame1">CANOPPY</li>
-                            <li><img class="dot" img src="css/img/Counter.png"></li>
+                            <li><img class="dot"  img src="css/img/Counter.png"></li>
                             <li class="frame2">Back End Dev</li>
                             <li><img class="dot" img src="css/img/Counter.png"></li>
                             <li class="frame3">2015</li>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] Please add an `alt` attribute to every `img` tag that's useful for interacting or understanding the page has an alt text, for example, all the portfolio project images have an alt text.
- [ ] If the image is not important for understanding the content — for example, it is just decoration or "eye candy" — it should have null alt `(alt="")`. One way to help determine if an image should have a `null alt` is to ask yourself: If the image was removed, would the user still get all the information from the page?

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @Kwamay, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Kwamay, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## To Highlight 👍

- Has a proper Pull Request. ✔️
- Portfolio project accessibility ✔️
- Color contrast ✔️
- Alternative text for images ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**